server/datastore/mysql/migrations/data/20210819120215_ChangeTeamScheduleNames.go

```diff
@@ -0,0 +1,24 @@
+package data
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20210819120215, Down_20210819120215)
+}
+
+func Up_20210819120215(tx *sql.Tx) error {
+	sql := `UPDATE packs SET name = CONCAT('Team schedule for: ', (select name from teams where id=(CAST(SUBSTRING_INDEX(pack_type,'-',-1) AS UNSIGNED)))) WHERE pack_type LIKE 'team-%'`
```

Comment:
```suggestion
	sql := `UPDATE packs SET name = CONCAT('Team: ', (select name from teams where id=(CAST(SUBSTRING_INDEX(pack_type,'-',-1) AS UNSIGNED)))) WHERE pack_type LIKE 'team-%'`
```

Reply:
Happy to make this change, but before doing so, pinging @noahtalerman since we discussed this format before.

Reply:
Oh! Can you link me that discussion please?

Reply:
Not a long discussion https://github.com/fleetdm/fleet/issues/1661#issuecomment-901971307 :)

Reply:
I think we can go ahead with my proposed change. I also updated that discussion to reflect this. Thanks!